doc/iterations/2024.3/shapes/add_support_for_browse_upload_images_widget.adoc

```diff
@@ -0,0 +1,95 @@
+= Add support for a browse / upload images widget
+
+== Problem
+
+Up until today, there is no specific widget to manage images.
+We use a selection widget that only displays image label and does not show any preview.
+It's not possible to add new images either.
+The only available images are the one programmatically loaded on the backend.
+
+== Key Result
+
+Have a single widget for both selecting an image (with an integrated preview) and contributing new images, either from a web link or from the local disk.
+
+== Solution
+
+Create a new custom widget in Sirius Components.
```

Comment:
What is the reason to add a new custom widget? Why not a regular one?

Reply:
I thought new widgets were now contributed as custom widgets.

Reply:
I don't remember we decided to do that, but please check that it should be the case, or not.

Reply:
After discussion, a custom widget will be developed.

Reply:
Why? I don't see why we should pay yet another additional cost to maintain a new widget. We have already two examples showing the API built for custom widgets, I don't see the need for another one. I don't see any benefit from going in this direction.

Reply:
I've changed it in the shape, it's now a core widget